_posts/2021-07-07-JdlC-meetup-June.md

```diff
@@ -0,0 +1,31 @@
+
+Open Science can look at the Free Software movement for an example on how to openly share its output. Free Software developers recognise the importance of their software&#39;s reusability, which means they understand the importance of licensing and diversity of licenses. Scientists, on the other hand, do not hold the power to put a license on their scientific output. De la Cueva argues that this should change. Science could, for example, take inspiration from the Internet Engineering Task Force (IETF), which manages standards for the Internet. Any contributor permits the IETF to distribute their contributions freely, and in turn can use any of the other standards, which have been licensed in the same way. This results in an open Internet that is freely accessible to all. If we want CUDOS for science, the same mechanism could be applied to scientific outputs, where scientists permit their contributions to be distributed freely, and in turn are able to freely use others&#39; contributions.
+
+De la Cueva points at two signs that science is moving in the right direction. First, the existence of preprint servers. Second, the [rise in peer-reviewed open access journals](https://www.aje.com/arc/open-access-changing-landscape/). And arguably, the increasingly common practice of publicly sharing research software, and adherence to FAIR data principles, can be counted as positive signs too. But there&#39;s still work to be done: &quot;_We still pay 9500 euros to obtain open access in_ Nature. _And the default for publications is still_ All rights reserved. _This has to end._&quot;
+
+We&#39;d like to thank Javier de la Cueva for his talk on Open Science. This talk was part of the June NL-RSE meetup. NL-RSE meetups are monthly events, where those working on the intersection between research and software development (from PhD students to Scientific Programmers) at Dutch institutes come together to discuss relevant topics. Our next NL-RSE meetup is on September 16 and will cover reproducible research articles. Speakers include Fernando Pérez, creator of the IPython programming environment, Giulia Guizzardi, Innovation Officer at eLife, and Adina Wagner, author of the software package Datalad. [More information and sign-up here](https://nl-rse.org/events/2021-09-15-meetup.html).
```

Comment:
Maybe link to the tweeter accounts and/or projects (Jupyter, eLife, Datalad, etc).

Reply:
> Left a few minor comments, but feel free to merge when you are ready.

I'm going to merge now, and will put adding the links / slides on my (longish) to-do list!